composer.json

```diff
@@ -29,7 +29,7 @@
         }
     ],
     "require": {
-        "php": ">=7.0",
+        "php": "^7.0|^8.0",
```

Comment:
You're right I made my PR too fast :-) I had two things in mind and my PR is the mix of both.

We have two options : 

- We could as proposed use `^7.0|^8.0`. In theory this should always work since PHP is not supposed to break any compatibility in minor version so if the code is compatible for 8.0 it should also be for 8.1. Most of the libs/apps are using this (composer itself is using this). If by any mean PHP break the compatibility I'm sure it would be possible to create a bug fix version to fix the issue.

- If we want to prevent 8.1 we should have `^7.0|~8.0` PHP sometime break small compatibility things in minor versions so this would be the safest but this is probably overkill.

Reply:
You are right regarding minor version compatibility. But there might be some newly introduced bugs, like it was with versions 7.4.0-7.4.1. I think version support should come together with CI builds. So let's keep it strictier.
BTW constraint `~8.0` allows to use on all versions up to `9.0`.

Reply:
I just forced push a new commit to replace the current one with `^7.0|~8.0.0` which THIS TIME should be correct for what you want !